cartridge/failover/raft.lua

```diff
@@ -129,6 +131,7 @@ local function cfg()
     end
 
     membership.set_payload('raft_term', box.info.election.term)
+    membership.set_payload('raft_leader', box.info.election.leader)
```

Comment:
`box.info.election.leader` is local number but need to store  uuid

Reply:
![image](https://user-images.githubusercontent.com/1633879/214334350-994fd255-294c-43e5-b262-f988665554f6.png)

Reply:
Also box.info.election.leader may be zero  during start of new election process

Reply:
I'm not sure, but it seems there is no guarantee that the same number must correspond to the same uuid on different instances

Reply:
> I'm not sure, but it seems there is no guarantee that the same number must correspond to the same uuid on different instances

On different instances box.info.election.leader is the same as numbers in box.space._cluster, so it corresponds to the same uuid if everything was ok in election process